packages/template-1c/app/components/hello-javascript.d.ts

```diff
@@ -0,0 +1,2 @@
+export default HelloJS;
+declare function HelloJS(): JSX.Element;
```

Comment:
One of the reasons I chose template-1c package for this PR because it has the .ts files. And it also has javascript like this file too. So we can see that the type declaration files are generated for both ts and js. That's because of the `allowJs` option enabled in the tsconfig.json.